src/test/java/staffconnect/model/meeting/MeetingTest.java

```diff
@@ -0,0 +1,62 @@
+package staffconnect.model.meeting;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+
+class MeetingTest {
+    private static final Description testDescription = new Description("Valid description");
+    private static final MeetDateTime testDate = new MeetDateTime("12/04/2023 12:00");
+    private static final Description otherDescription = new Description("Another valid description");
+    private static final MeetDateTime otherDate = new MeetDateTime("13/04/2023 13:00");
+
+    @Test
+    public void equals() {
+
+        Meeting testMeeting = new Meeting(testDescription, testDate);
+        Meeting diffMeetingDescription = new Meeting(otherDescription, testDate);
+        Meeting diffMeetingTime = new Meeting(testDescription, otherDate);
+        MeetDateTime testDate = new MeetDateTime("12/04/2023 12:00");
```

Comment:
I think in future iterations, this could be extracted out, similar to `Person` -> `ALICE`

Reply:
Yup it is a small test case for now, thats why I left it in to keep things simple